mne/viz/evoked.py

```diff
@@ -126,6 +127,36 @@ def _rgb(x, y, z):
     return np.asarray([x, y, z]).T
 
 
+def _plot_legend(pos=None, colors=None, axis=None, outlines='head'):
```

Comment:
And I wonder if we even need the ``outlines`` param here. It doesn't seem to work very well in the legend.